logging.rb

```diff
@@ -0,0 +1,25 @@
+require 'logger'
+
+module Logging
+  class << self
+    def logger
+      @logger ||= Logger.new STDOUT
+    end
+
+    def logger=(logger)
+      @logger = logger
+    end
+  end
+
+  def self.included(base)
+    class << base
+      def logger
+        Logging.logger
+      end
+    end
+  end
+
+  def logger
+    Logging.logger
+  end
+end
```

Comment:
@pfarestveit This module does seem kinda snarled. I'm curious whether a simpler module of the sort used by CalCentral's class logger would work:

https://gist.github.com/paulkerschen/e4b4d2d58e503dbf77e1802301ceba9d

and if it doesn't work, I'm curious why not.